scripts/spack_packages/packages/care/package.py

```diff
@@ -40,7 +40,7 @@ class Care(CachedCMakePackage, CudaPackage, ROCmPackage):
     depends_on('cmake@3.14.5:', type='build')
     depends_on('cmake@3.21:', when='+rocm', type='build')
 
-    depends_on('blt@0.5.2:', type='build', when='@0.10.0:')
+    depends_on('blt@0.5.2', type='build', when='@0.10.0:')
```

Comment:
I'm not familiar with the spack syntax.  What does the colon mean?

Reply:
I think `blt@0.5.2:` would mean that spack can use blt 0.5.2 or greater (at least that version), while without the colong we we fixed to version 0.5.2 exactly